alf/examples/actor_critic.py

```diff
@@ -72,22 +72,20 @@
 import tensorflow as tf
 
 from tf_agents.environments import parallel_py_environment
-from tf_agents.environments import suite_mujoco
 from tf_agents.environments import suite_gym
 from tf_agents.environments import tf_py_environment
 from tf_agents.networks.actor_distribution_network import ActorDistributionNetwork
 from tf_agents.networks.actor_distribution_rnn_network import ActorDistributionRnnNetwork
 from tf_agents.networks.encoding_network import EncodingNetwork
 from tf_agents.networks.value_network import ValueNetwork
 from tf_agents.networks.value_rnn_network import ValueRnnNetwork
-from tf_agents.environments import atari_wrappers
 
 from alf.algorithms.actor_critic_algorithm import ActorCriticAlgorithm
 from alf.algorithms.icm_algorithm import ICMAlgorithm
 from alf.drivers.on_policy_driver import OnPolicyDriver
 from alf.environments import suite_socialbot
-from alf.environments import suite_mario
```

Comment:
shoud add corresponding import statement in gin config file since this line here been removed